Build failing on Python 2.7 and 3.4

The [latest build](https://travis-ci.com/mwouts/jupytext/builds/138765451) fails for Python 2.7 and 3.4. The test in error is 
```
test_ipynb_to_pandoc[/home/travis/build/mwouts/jupytext/tests/notebooks/ipynb_julia/julia_benchmark_plotly_barchart.ipynb]
```
and the error is that the YAML header has changed slightly:

```
E           --- expected
E           +++ actual
E           @@ -1,7 +1,7 @@
E            ---
E            jupyter:
E              kernelspec:
E           -    display_name: 'Julia 1.1.1'
E           +    display_name: Julia 1.1.1
E                language: julia
E                name: 'julia-1.1'
E              nbformat: 4
```

--- a/tests/notebooks/mirror/ipynb_to_pandoc/julia_benchmark_plotly_barchart.md
+++ b/tests/notebooks/mirror/ipynb_to_pandoc/julia_benchmark_plotly_barchart.md
@@ -1,7 +1,7 @@
 ---
 jupyter:
   kernelspec:
-    display_name: 'Julia 1.1.1'
+    display_name: Julia 1.1.1
     language: julia
     name: 'julia-1.1'
   nbformat: 4

